### PR TITLE
Deck Button Grid: reorder

### PR DESCRIPTION
**Pull request by NotYourAverageAl**
interface: reorder deck grid buttons to align with screenshot

### CODE REVIEW

**Comment by ronso0** [comment]
Thanks for fixing this.
However we should describe the controls in this order:
top row, left>right
bottom row, left>right

so it should be 
* Slip Mode
* Quantize
* Beatgrid Adjust
* **Eject**
* Repeat
* **Keylock**

File: source/chapters/user_interface.rst [hunk 1]
```diff
@@ -1074,22 +1075,23 @@ Track Information Display
...
   :term:`Keylock <key lock>` locks the track's pitch so that adjustments to its
   tempo via the rate slider do not affect its pitch.
 
-**Mix Orientation Toggle**
-  This control determines the sampler input's mix orientation.
-
-**Eject Track Button**
-  Clicking this button ejects the track from the deck.
+**Crossfader Orientation Toggle**
+  The :term:`crossfader` orientation determines to which side of the crossfader the deck is assigned.
```
Comment (ronso0) [inline source/chapters/user_interface.rst:1093]:
```suggestion
  The :term:`crossfader` orientation determines to which side of the crossfader the sampler is assigned.
```
The section heading should also be "~~Deck~~ Sampler Options Button Grid"

**Comment by ronso0** [comment]
one more comment.
after fixing that can you please squash the commits so that we only two: one for decks and one for samplers.
Thank you.

**Comment by NotYourAverageAl** [comment]
god damn git @ronso0 how do I delete the last two merge & revert commits

**Pull request merged**